deployments/mainnet/weth/migrations/1718698838_add_wbtc_as_collateral.ts

```diff
@@ -0,0 +1,157 @@
+import { expect } from 'chai';
+import { DeploymentManager } from '../../../../plugins/deployment_manager/DeploymentManager';
+import { migration } from '../../../../plugins/deployment_manager/Migration';
+import { exp, proposal } from '../../../../src/deploy';
+
+
+const WBTC_ADDRESS = '0x2260FAC5E5542a773Aa44fBCfeDf7C193bc2C599';
+const WBTC_BTC_PRICE_FEED_ADDRESS = '0xfdFD9C85aD200c506Cf9e21F1FD8dd01932FBB23';
+const BTC_ETH_PRICE_FEED_ADDRESS = '0xdeb288F737066589598e9214E782fa5A8eD689e8';
+
+export default migration('1718698838_add_wbtc_as_collateral', {
+  async prepare(deploymentManager: DeploymentManager) {
+    const _wbtcScalingPriceFeed = await deploymentManager.deploy(
```

Comment:
Add comment about WBTCPriceFeed contract. It is used for WBTC/ETH, but also can be used for WBTC/USD